### PR TITLE
Readme update

### PR DESCRIPTION
**Pull request by kashishkebab9**
Just includes some stuff for future scarab admins, more specific instructions, some troubleshooting steps. Ignore how many commits there are, was having issues with setting up github on my machine.

### CODE REVIEW

File: doc/Admin.md [hunk 1]
```diff
@@ -2,7 +2,16 @@
 
 Install Ubunutu 20.04. Choose not to connect to the internet during setup and choose the minimal installation option. Make an admin user `scarab_admin` with hostname `scarab<id>`, replacing `<id>` with the scarab's ID. Throughout these instructions, all instances of `<id>` should be replaced with the scarab's ID.
 
-After restarting and removing the installation medium, configure a connection to the scarab network using NetworkManager with a manual IP address of `192.168.131.<id>`, Netmask of `255.255.255.255`, Gateway of `192.168.131.1`, and DNS of `192.168.131.1`. Once connected, you can verify the settings were successful by viewing the Details tab of the scarab network settings in NetworkManager or manually on the command line by running `ip address show` to check the right IP address was assigned and `ping www.google.com` to verify internet access.
+After restarting and removing the instaalion media, connect to the 'scarab' wifi network (get password from lab admins). Go to Settings-->Wi-Fi and click on the gear icon next to the scarab network. From there go to IPv4 and set the IPv4 Method to Manual. Enter the following information:
```
Comment (danmox) [inline doc/Admin.md]: fix typo

File: doc/Admin.md [hunk 2]
```diff
@@ -2,7 +2,16 @@
...
+
+IP address: `192.168.131.<id>`\
+Netmask: `255.255.255.255`\
+Gateway: `192.168.131.1`
+
+Turn off the Automatic toggle next to DNS and enter the following information:\
+DNS: `192.168.131.1`
+
+Once connected, you can verify the settings were successful by viewing the Details tab of the scarab network settings in the same windo or manually on the command line by running `ip address show` to check the right IP address was assigned and `ping www.google.com` to verify internet access. If you are having issues connecting to the internet, verify the above information and perform a reboot, this is a common issue to see.
```
Comment (danmox) [inline doc/Admin.md]: fix typo

Comment (danmox) [inline doc/Admin.md]: After applying the settings and before restarting, executing:
```sudo systemctl restart NetworkManager.service```
may solve the connection issue.

Comment (kashishkebab9) [reply doc/Admin.md]: Added workaround, if problem persists after restarting the service, only then should the user reboot

**Pull request merged**